### PR TITLE
feat: create multiple namespaces with namespace_ids, one namespace per scope …

### PR DESCRIPTION
**Pull request by yliaog**
feat: create multiple scopes with scope_ids, one namespace per scope with the same name

### CODE REVIEW

**Comment by apeabody** [comment]
re-triggered test

**Review by apeabody — APPROVED** [review]
Thanks @yliaog!

File: 4-fleetscope/modules/env_baseline/variables.tf [hunk 1]
```diff
@@ -24,14 +24,9 @@ variable "fleet_project_id" {
...
-  description = "The fleet scope ID"
-  type        = string
-}
-
-variable "namespace_id" {
-  description = "The fleet namespace ID"
-  type        = string
+variable "scope_ids" {
+  description = "The fleet scope IDs"
+  type        = list(string)
```
Comment (gtsorbo) [inline 4-fleetscope/modules/env_baseline/variables.tf:29]: Can we use `namespace_ids` as the variable here instead? It makes it clearer that we are creating the namespaces at this stage, and creating a scope for each namespace

**Review by gtsorbo — CHANGES_REQUESTED** [review]

File: 4-fleetscope/modules/env_baseline/variables.tf (hunk 1 above)
Comment (yliaog) [reply 4-fleetscope/modules/env_baseline/variables.tf]: sure, rename to namespace_ids

**Review by gtsorbo — APPROVED** [review]

**Pull request merged**